### PR TITLE
6 view individual article

### PR DESCRIPTION
**Pull request by majidfer**
Completed ticket #6 View an individual article

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        setArticle(articleFromApi.article);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setError(err.response.data);
+      });
+  }, [article_id]);
+
+  if (isError) {
+    return <p className="error">Oopps.. some error occured!!</p>;
```
Comment (leekli) [inline src/components/Article.jsx]: Could you make this error message a bit more specific e.g. Error 404 - that article does not exist/not found?

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -32,7 +41,11 @@ function Articles() {
             return (
               <li key={article.article_id} className="article-card">
                 <article>
-                  <h2>{article.title}</h2>
+                  <h2>
+                    <Link to={`/articles/${article.article_id}`} target="_top">
```
Comment (leekli) [inline src/components/Articles.jsx]: Please can you remove target="_top" as it's making it not work properly?

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Ferial, a good submission.

I've added 2 comments, please could you look at these before committing/merging?

**Review by popatre — APPROVED** [review]
Looks all good

**Pull request merged**